### PR TITLE
WIP: fix: remove space from marker annotations

### PR DESCRIPTION
**Pull request by camilamacedo86**
Closes: #1384

TODO:
- Allow adding fragments when has the marker no matter if has or not an space between the comment // and marker.

### CODE REVIEW

**Comment by camilamacedo86** [comment]
c/c @Adirio

File: pkg/model/file/marker.go [hunk 1]
```diff
@@ -24,9 +24,9 @@ import (
 const prefix = "+kubebuilder:scaffold:"
 
 var commentsByExt = map[string]string{
-	// TODO(v3): machine-readable comments should not have spaces by Go convention. However,
-	//  this is a backwards incompatible change, and thus should be done for next project version.
-	".go":   "// ",
+	// it is used just to gen the marker annotations (//+)
+	".go": "//",
+	// It is used just to gen the marker commanets in the yaml files (# +)
 	".yaml": "# ",
```
Comment (Adirio) [inline pkg/model/file/marker.go:30]: I would re-write the comments inline to keep 1 line per file type and including a longer example:
```suggestion
	".go":   "//", // Marker annotations for go files should not have spaces by go convention: //+kubebuilder:*
	".yaml": "# ", // Marker annotations for yaml files are more readable with spaces: # +kubebuilder:*
```

**Comment by camilamacedo86** [comment]
HI @Adirio, 

Remove an space of the annotations which are scaffolded do not change the layout of the projects and is not a breaking change. Projects scaffolded before and after this change still working with the tool and then, it is not a breaking change at all.

**Comment by camilamacedo86** [comment]
/hold
The problem here is that annotation without space will not found to add new fragments.
Users also could change it so we need to allow it works with space or not.

**Comment by camilamacedo86** [comment]
I am closing this one. I think better about this change and I do not think that we should move forward with.

**Pull request closed without merging**